versions.props

```diff
@@ -13,7 +13,7 @@ com.palantir.remoting3:* = 3.46.0
 com.palantir.safe-logging:* = 1.10.1
 com.palantir.tracing:* = 3.1.0
 com.squareup.okhttp3:okhttp = 4.0.0
-com.uber.nullaway:nullaway = 0.6.4
+com.uber.nullaway:nullaway = 0.7.5
```

Comment:
Have also added this to excavator-versions-props so you should get this automatically next time :)

Reply:
Thanks!